bindings/node/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required (VERSION 3.12)
```

Comment:
@YakoYakoYokuYoku Starting to have a look at this. Are there specific CMake features that are being used here that require this version? Our Evergreen build images currently have Cmake 2.8.12.2 installed so they currently have errors on the version check.

Reply:
Root `CMakeLists.txt` uses this version, so that's the reason why I've used this one.

https://github.com/mongodb/libmongocrypt/blob/2f48f35f460423b6dbd7b53125861c93096fce0e/CMakeLists.txt#L1

Also isn't `2.8.12.2` a bit too old?

Reply:
Yeah it's old, but that's what our some of our CI images have. I'll need to open an internal ticket to get those updated - thanks so far for the work.